Library/Models/InterviewerAssignmentModel.cs

```diff
@@ -0,0 +1,97 @@
+//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+
+namespace Nfield.Models
+{
+    /// <summary>
+    /// Model for interviewer assignment
+    /// </summary>
+    public class InterviewerAssignmentModel
+    {
+        /// <summary>
+        /// Survey name
+        /// </summary>
+        public string SurveyName { get; set; }
+        /// <summary>
+        /// Survey Id
+        /// </summary>
+        public string SurveyId { get; set; }
+        /// <summary>
+        /// Interviewer user  name
+        /// </summary>
+        public string Interviewer { get; set; }
+        /// <summary>
+        /// Interviewer Id
+        /// </summary>
+        public string InterviewerId { get; set; }
+        /// <summary>
+        /// Discriminator
+        /// </summary>
+        public string Discriminator { get; set; }
+        /// <summary>
+        /// Assigned or not
+        /// </summary>
+        public bool? IsAssigned { get; set; }
```

Comment:
Changing all these items to nullable can generate a lot of issues also with existing software. Is it completely required? Why? (we really need a clear reason for this, otherwise we should use different models)